package.json

```diff
@@ -196,7 +196,7 @@
     "build-client": "cross-env-shell CALYPSO_CLIENT=true NODE_PATH=$NODE_PATH:server:client:. node --max_old_space_size=8192 ./node_modules/webpack/bin/webpack.js --display-error-details --config webpack.config.js",
     "build-client-if-prod": "node -e \"process.env.CALYPSO_ENV === 'production' && process.exit(1)\" || npm run build-client",
     "build-client-if-desktop": "node -e \"process.env.CALYPSO_ENV === 'desktop' && process.exit(1)\" || npm run build-client",
-    "build-block": "node bin/create-scripts/block.js",
+    "build-block": "SKIP_FLAG_IMAGES=true node bin/create-scripts/block.js",
```

Comment:
In `build-server` and `build-client` we're using `cross-env-shell` when setting env vars — I don't know if we need to support Windows here but I'd follow that example.

Reply:
Good call, updated in https://github.com/Automattic/wp-calypso/pull/26260/commits/ce56d4d36c6e10b9c17596d1bbb8b2ad572a3fe7